libsrc/cbm/cbm_read.s

```diff
@@ -45,11 +45,11 @@
 
 
 _cbm_read:
-        eor     #$FF
-        sta     ptr1
-        txa
-        eor     #$FF
-        sta     ptr1+1          ; Save -size-1
+        inx
+        stx     ptr1+1
+        tax
+        inx
+        stx     ptr1            ; Save size with each byte incremented.
```

Comment:
Should this say something like "Save size, *decremented* on each byte"?

Reply:
No, both are incremented with `inx` before they are stored in `ptr1+1` and `ptr1`. The comment is about what was done to the values, not about what happens with them afterwards.

Reply:
I think that the phrase is slightly confusing.  I recommend, "with both bytes incremented separately".

Reply:
@greg-king5 Sorted.